src/wallet.h

```diff
@@ -111,13 +112,15 @@ class CWallet : public CCryptoKeyStore, public CWalletInterface
     CWallet()
     {
         SetNull();
+        nStakeSplitThreshold = 10000000;
```

Comment:
so by default this means that stakes will split after 10 million?

Reply:
that is correct. we can move the number to something lower if needed. i just put in some defualt

Reply:
If we are not augmenting the code when patching it in from another wallet it won't split at 10 million. What will happen (based on video demonstrations from other coins) is it will split at any number below 10 Million, so starting with 8 Million you will split into 2x4Million+reward inputs, and again down to 4x2Million. If your over 10 Million it prevents splitting and will never split again.

This is why I have posted into the Development channel questions about if we want to just inherit this logic since it just creates an on/off switch instead of setting a ceiling at which you will again split inputs. I think more control over 'when' to split would be useful and go along with the original concepts behind input splitting better than a simple off switch.